server/application.go

```diff
@@ -36,11 +38,14 @@ func NewApp(cfg config.AppConfig, db *pgxpool.Pool) *App {
 		IdleTimeout:  15 * time.Second,
 	}
 
+	queries := sqlc.New(db)
+
```

Comment:
Maybe it might be better to instantiate this inside `database/connection.go`, as soon as the connection is established? Just for organization's sake; I kind of meant the `NewApp` function to just return an `App*` instance and do nothing other than that